packages/near-api-js/test/wallet-account.test.js

```diff
@@ -154,12 +155,63 @@ it('can complete sign in', async () => {
     await walletConnection._completeSignInWithAccessKey();
 
     expect(await keyStore.getKey('networkId', 'near.account')).toEqual(keyPair);
-    expect(localStorage.getItem('contractId_wallet_auth_key'));
+    expect(localStorage.getItem('contractId_wallet_auth_key')).toBeTruthy();
     expect(history.slice(1)).toEqual([
         [{}, 'documentTitle', 'http://example.com/location']
     ]);
 });
 
+it('can get the default appKeyPrefix', async () => {
+    nearFakeWithoutContract = {
+        config: {
+            networkId: 'networkId',
+            walletUrl: 'http://example.com/wallet',
+        },
+        connection: {
+            networkId: 'networkId',
+            signer: new nearApi.InMemorySigner(keyStore)
+        },
+        account() {
+            return {
+                state() { }
+            };
+        }
+    };
+
+    const keyPair = nearApi.KeyPair.fromRandom('ed25519');
+    global.window.location.href = `http://example.com/location?account_id=near.account&public_key=${keyPair.publicKey}`;
```

Comment:
Can we add `all_keys` to the url here and make equality assertion on L188?

```suggestion
    global.window.location.href = `http://example.com/location?account_id=near.account&public_key=${keyPair.publicKey}&all_keys=${keyPair.toString()}`;
```

Then we can do: 

```js
expect(localStorage.getItem('default_wallet_auth_key')).toEqual(JSON.stringify({accountId: 'near.account', allKeys: [keyPair.toString()]}));
```